app/components/UI/Stake/Views/StakeInputView/StakeInputView.tsx

```diff
@@ -76,6 +77,13 @@ const StakeInputView = ({ route }: StakeInputViewProps) => {
     });
   };
 
+  const [isSubmittingStakeDepositTransaction, setIsSubmittingStakeDepositTransaction] = useState(false);
+  useFocusEffect(
+    useCallback(() => {
+      setIsSubmittingStakeDepositTransaction(false);
+    }, [])
+  );
+
```

Comment:
Nit: Vertical spacing

```suggestion
  const [isSubmittingStakeDepositTransaction, setIsSubmittingStakeDepositTransaction] = useState(false);
  
  useFocusEffect(
    useCallback(() => {
      setIsSubmittingStakeDepositTransaction(false);
    }, [])
  );

```